### DISCUSSION TITLE
Returned Metadata (UMM?)

### DISCUSSION THREAD

**Discussion by scottyhq** [earthaccess-dev/earthaccess#213] [General]
I'm not too familiar with all the possible CMR metadata values, but am surprised when granules come back from a search without a footprint polygon. It looks like earthaccess requires umm_json:
https://github.com/nsidc/earthaccess/blob/3a2d171dbe3ca4348fedb3b24c3a3c31ed2bf133/earthaccess/search.py#L18

But it seems like that doesn't return the polygon... See for example:
curl "https://cmr.earthdata.nasa.gov/search/granules.json?collection_concept_id=C2120512202-NSIDC_ECS&producer_granule_id\[\]=ATL03_20221012161617_03351705_005_01.h5" 

versus
curl "https://cmr.earthdata.nasa.gov/search/granules.umm_json?collection_concept_id=C2120512202-NSIDC_ECS&producer_granule_id\[\]=ATL03_20221012161617_03351705_005_01.h5" 

Two questions come to mind:
1. What is umm_json and is this a bug that there is no polygon for the above result?
2. I couldn't figure out how to replicate the above query with `earthaccess.search()` (If I know a granule name in advance)

**Comment by betolink** [comment]
Yes, I think this is a bug in our side. This also came up with https://github.com/evetion/SpaceLiDAR.jl/pull/29 and I think we should fix it. I don't know if it will happen soon or ever so we better work on refactoring the result classes so they can parse at least the simple JSON flavor from CMR. The main reason I picked UMM was that it's usually more reliable and complete but as we can see here this is not always the case. @scottyhq
